README.md

```diff
@@ -561,6 +561,13 @@ timescaledb:
   # database: "" # TimescaleDB database used
   # hypertablename: "falco_events" # Hypertable to store data events (default: falco_events) See TimescaleDB setup for more info
   # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
+redis:
+  # host: "" # Redis host, if not empty, Redis output is enabled
+  # port: "6379" # TimescaleDB port (default: 5432)
+  # password: "" # Password to authenticate with Redis (default: "")
+  # database: "" # Redis database number (default: 0)
+  # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
+
```

Comment:
what about to allow to set a  `ttl` for keys?

Reply:
@Issif , I am not sure if that can be done through redis config.